### PR TITLE
Unittest to pytest

### PR DESCRIPTION
**Pull request by atamazian**
Implements https://github.com/iver56/audiomentations/issues/176, except from `unittest.mock` in `test_room_simulator.py`. I can replace it with `pytest-mock` if necessary.

### CODE REVIEW

**Comment by atamazian** [comment]
Can you ignore first two commits? They're related to https://github.com/iver56/audiomentations/pull/192 , and by mistake I included them here.

**Comment by iver56** [comment]
Could you somehow rebase out those commits from this branch? It would make it easier to get in this PR

**Comment by atamazian** [comment]
Done! Can you review this now?

**Comment by iver56** [comment]
I tried to run the test suite, but it couldn't start due to:

```
___________________________________________________________ ERROR collecting tests/test_utils.py ___________________________________________________________
ImportError while importing test module 'C:\Users\iver5\Code\audiomentations\tests\test_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests\test_utils.py:4: in <module>
    from audiomentations.core.utils import (
E   ImportError: cannot import name 'find_audio_files_in_paths' from 'audiomentations.core.utils' (C:\Users\iver5\Code\audiomentations\audiomentations\core\utils.py)
```

**Comment by atamazian** [comment]
Fixed. Can you check it now?

File: tests/test_gaussian_snr.py [hunk 1]
```diff
@@ -43,6 +43,5 @@ def test_gaussian_noise_snr_multichannel(self):
         samples_out = augmenter(samples=samples, sample_rate=16000)
 
         assert samples_out.dtype == np.float32
-        self.assertGreater(
-            float(np.sum(np.abs(samples_out))), float(np.sum(np.abs(samples)))
-        )
+        assert float(np.sum(np.abs(samples_out))) > float(np.sum(np.abs(samples_out)))
```
Comment (iver56) [inline tests/test_gaussian_snr.py:46]:
```suggestion
        assert float(np.sum(np.abs(samples_out))) > float(np.sum(np.abs(samples)))
```

**Review by iver56 — CHANGES_REQUESTED** [review]
Looks good to me, except this small error (the test fails)

**Comment by atamazian** [comment]
Should I leave `unittest.mock` in `test_room_simulator.py`? Or it's better to replace it with something else?

**Comment by iver56** [comment]
> Should I leave `unittest.mock` in `test_room_simulator.py`? Or it's better to replace it with something else?

You can leave it as it is for now 👍

**Review by iver56 — APPROVED** [review]

**Comment by iver56** [comment]
Thanks for the contribution!

**Pull request merged**